.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,153 +1,85 @@
 name: Unit, Smoke, Connected and Integration tests
-env:
-  K8_CLUSTER: 'integration-master'
-  # This must be defined for the bash redirection
-  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
-  # This must be defined for the bash redirection
-  GOOGLE_SA_CERT: 'jade-dev-account.pem'
-  # This will make test logs not use JSON
-  TDR_LOG_APPENDER: 'Console-Standard'
-  # The default Azure credentials to use to authenticate
-  AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
-  AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
-  JADE_USER_EMAIL: connected-tdr-user@notarealemail.org
-  RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
-  AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-  AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
-  AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
-  AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
-  CACHE_PATHS: |
-    build/jacoco
-    build/reports
-    build/spotless
-    build/test-results
-    build/jacocoHtml
 on:
-  pull_request:
-    branches:
-      - develop
-    paths:
-      - '!*'
-      - 'src/**'
-      - 'gradle**'
-      - 'gradle/**'
-      - '**.gradle'
-      - 'Dockerfile'
-      - 'datarepo-clienttests/**'
-      - '.github/workflows/int-and-connected-test-run.yml'
-      - '.swagger-codegen-ignore'
   workflow_dispatch: {}
+  pull_request:
```

Comment:
What does an empty `pull_request` block do? I can't find any docs specifically on this.

Reply:
It means it will run when a `pull_request` is submitted.